geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloFeatureWriter.scala

```diff
@@ -142,8 +117,7 @@ abstract class AccumuloFeatureWriter(sft: SimpleFeatureType,
       feature
     }
 
-    val toWrite = new FeatureToWrite(withFid, defaultVisibility, encoder, indexValueEncoder)
-    writers.foreach(write => write(toWrite))
+    writer(new FeatureToWrite(withFid, defaultVisibility, encoder, indexValueEncoder))
```

Comment:
Yeah, it is another object. The alternatives seem bad though - make the writer take the encoded feature values (which causes duplicated code and they all might not be used in every case), or break up the writer interface into specific impls for each table. What do you think?